### DISCUSSION TITLE
How to get headers from Airflow REST API?

### DISCUSSION THREAD

**Discussion by FDU-Laplace** [apache/airflow#26388] [Q&A] [answered]
When I triggered an Airflow DAG by REST API
`dags/{dag_id}/dagRuns`
Then I can easily get the body of the trigger request by using `"{{dag_run.conf}}"`.
How can I get the headers of the trigger request?

**Comment by potiuk** [comment, marked as answer]
I think you can't. You should pass whatever you want via dag_run.conf. If you have a good reason why this is not possible, you are free to create a PR adding this feature (with appropriate justification why you want to add it.

You can become one of  > 2100 contributors to Airlfow.

**Reply by FDU-Laplace** [reply]
Thanks for your reply!
